content/post/2017-09-15-friday-hacks-137-Sep-15.md

```diff
@@ -0,0 +1,29 @@
+---
+title: "Friday Hacks #137, September 15"
+date: 2017-09-12 09:40:35.299431
+author: Herbert
+---
+
+This Friday, our very own coreteam members will be haring their internship experiences both locally and overseas. We will also invite NUS students who have done internships in various companies.
+
+{{% friday_hack_header venue="Seminar Room 1, NUS School of Computing (COM1-0206)" date="September 15" %}}
+
+### Internships, and How to Get Them
+
+#### Talk Description:
+
+Learn why you should do internships, how to choose, and of course, how to get them!
+
+Ask questions of other students who have interned locally and overseas at various companies. This includes overseas like Google, Facebook, Twitter, Asana, and local companies like Google Singapore, Twitter Singapore, Carousell, Garena, and Tinkerbox!
```

Comment:
Ask questions of other students who have interned both locally and overseas. These companies include Google, Facebook, Twitter, Asana, as well as local companies like Google Singapore, Twitter Singapore, Carousell, Garena, and Tinkerbox!

Reply:
Add Apple as well